tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/CryptographyAlgorithm/SymmetricAlgorithmIntegrationCommon.cs

```diff
@@ -0,0 +1,44 @@
+// <copyright file="SymmetricAlgorithmIntegrationCommon.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+using System;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.Iast;
+using Datadog.Trace.Logging;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.CryptographyAlgorithm
+{
+    internal class SymmetricAlgorithmIntegrationCommon
+    {
+        internal const IntegrationId IntegrationId = Configuration.IntegrationId.SymmetricAlgorithm;
+        private static readonly IDatadogLogger Log = DatadogLogging.GetLoggerFor(typeof(SymmetricAlgorithmIntegrationCommon));
+
+        internal static Scope? CreateScope<TTarget>(TTarget instance)
+        {
+            if (instance is System.Security.Cryptography.SymmetricAlgorithm algorithm)
```

Comment:
we're instrumenting the `SymmetricAlgorithm` constructor, so this will _always_ be `SymmetricAlgorithm` right? I think we can remove the `if`.

Reply:
Done!